### DISCUSSION TITLE
Forum: Move topic

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#655] [Ideas]
For admin to have the ability to move a forum topic from one folder to another.

**Comment by televandalist** [comment]
Of 1707 "Official Topics" in the Mega Drive/Genesis forum, only 449 of them are actually for Mega Drive/Genesis games. Mega Drive/Genesis, SNES, NES, GB/GBC/GBA, and PC Engine are the only consoles that proper forum topics can be created in. The rest are created in the Mega Drive/Genesis forum by default. This is extremely messy 😩 I've already made a list of all topics in the Mega Drive/Genesis forum which need to be moved (posted on Discord).

An update to issue #8 - Since there are entries for all of these consoles, it would make sense for them to have their own forums so future topics can be created in the right place:
- 32X
- 3DO
- Apple II
- Arcade
- Atari 2600
- Atari 5200
- Atari 7800
- Atari Jaguar
- Atari Lynx
- Cassette Vision
- ColecoVision
- Commodore 64
- DOS (not yet supported, but a topic exists)
- Events
- Game Gear
- Hubs
- Intellivision
- Master System
- Neo Geo Pocket
- Nintendo 64
- Nintendo DS
- Oric
- PC-8000/8800
- PC-98
- PC-FX
- PlayStation
- PlayStation Portable
- Pokemon Mini
- Saturn
- Sega CD
- SG-1000
- Super Cassette Vision
- Vectrex
- Virtual Boy
- WonderSwan

I guess ideally, every console ID should have a forum attached to it so it's not something that needs to be repeatedly addressed. This was likely put on hold for issue #146. But at the rate topics are made, especially considering all the newly supported consoles, this is something that will just keep getting worse.

**Reply by ScottFromDerby** [reply]
...sorry. Hanging my head in shame over here
